### ISSUE TITLE
Appium not identifying slider in UWP windows application

### ISSUE THREAD

**Issue by krevathi1912** [microsoft/WinAppDriver#1082]
We have UWP application, we are automating it using appium. Application has a left navigation slider which is not getting identified by appium. I am beginner in appium & UWP automation. Any help is greatly appreciated

**Comment by anunay1** [comment]
Why do u want to use slider, APPIUM has auto scroll feature so it will automatically scroll to find the element.

**Comment by krevathi1912** [comment]
Slider i mean to say tab, that has many elements. On clicking the element opens a editor dialog. Appium is not identifying the tab, so not able to click on the element.

**Comment by anunay1** [comment]
Is inspect able to identify the tab

**Comment by krevathi1912** [comment]
No, inspect also not identify the tab

**Comment by anunay1** [comment]
Then Appium will not work, it works only on the elements identified by inspect.

**Comment by krevathi1912** [comment]
Anyother alternate solution?

**Comment by anunay1** [comment]
Ask your development team to give and id for the tab so that inspect is able to find the tab.

**Comment by krevathi1912** [comment]
ok thank you, i will check with them

**Comment by naeemakram** [comment]
Have you tried to use Win App Driver UI Recorder to inspect the same element?

**Comment by krevathi1912** [comment]
Yeah, I tried, but WinAppDriver UI Recorder is getting hanged when used for our UWP app
